pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -27,6 +27,7 @@ import (
 
 // ClusterServiceBroker represents an entity that provides
 // ClusterServiceClasses for use in the service catalog.
+// +k8s:openapi-gen=x-kubernetes-print-columns:custom-columns=NAME:.metadata.name,URL:.spec.url,STATUS:.status.conditions[*].reason
```

Comment:
Not sure on the formulation of `STATUS` here, how about remove it and we'll do it in another PR?